server/src/room.ts

```diff
@@ -66,7 +68,23 @@ export default class Room {
         }
     }
 
+    reset() {
```

Comment:
Shouldn't this method be in the Player class?

Reply:
it needs to reset fields on the room itself (gameStarted, playersFinished). we're resetting both the room and the players within the room

Reply:
Oh right. Well I guess you can put the resetting player scores in the Player class, but maybe a minor thing not needed to worry about.

Reply:
sure, I'll add a player reset function